docusaurus/docs/Flutter/02-customization/01-custom-widgets/04-adding_custom_attachments.mdx

```diff
@@ -62,22 +62,21 @@ First, we add a button which when clicked fetches and shares location into the `
 StreamMessageInput(
   actions: [
     InkWell(
-      child: Icon(
+      child: const Icon(
         Icons.location_on,
-        size: 20.0,
-        color: StreamChatTheme.of(context).colorTheme.grey,
+        size: 20,
+        color: Colors.grey,
```

Comment:
The colour fetch from theme was initially meant to cover for dark theme (although "grey" was definitely a bad name which has since been changed). Since this is for a simple custom attachment we can probably replace this with a solid colour but the grey has probably replaced with `textLowEmphasis` in the colour theme. Just a note, no change needed.

Reply:
I couldn't find the replacement, thanks!